app/components/primer/spinner_component.html.erb

```diff
@@ -1,7 +1,15 @@
-<span role="status">
+<%= render Primer::BaseComponent.new(**@system_arguments) do %>
   <span class="sr-only">Loading</span>
-  <%= render Primer::BaseComponent.new(**@system_arguments) do %>
+  <%= render Primer::BaseComponent.new(
+    tag: :svg,
+    viewBox: '0 0 16 16',
+    fill: 'none',
+    animation: :rotate,
+    width: @size,
+    height: @size,
```

Comment:
I was worried that there'd be instances where `style` was passed to override the spinner width/height but scanning through .com `style` is only used in one place and it's to set a `top` value